rust/rope/src/delta.rs

```diff
@@ -243,6 +244,31 @@ impl<N: NodeInfo> Delta<N> {
             }
         )
     }
+
+    /// Iterates over all the inserts of the delta. Each insert is represented by a tuple of
+    /// three elements, where the first element represents the position of the insertion relative
+    /// to the old document, the second element the position relative to the new document and the
+    /// third element the length of the insertion.
```

Comment:
nit: this seems like a sort of weird/esoteric representation. I might in any case be inclined towards defining some `DeltaOpSummary` struct that gives these values useful names? `old_offset, new_offset, len` or something. Not sure if that improves ergonomics in any practical way, or not...

Anyway I'm just thinking out loud, you should ignore unless you personally think this would be an improvement.